dnnlib: Running training.training_loop_3d.training_loop() on localhost...
WARNING:tensorflow:From /n/home09/scajasordonez/anaconda3/envs/3dgan_tf2/lib/python3.8/site-packages/tensorflow/python/compat/v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
CUDA CASHE PATH
/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/_cudacache
Streaming data using training.dataset.TFRecordDataset...
(1, 5, 6, 7)
(1, 10, 12, 14)
(1, 20, 24, 28)
(1, 40, 48, 56)
(1, 80, 96, 112)
(1, 160, 192, 224)
=================================================
 Dataset Repeated
=================================================
=================================================
 Dataset Repeated
=================================================
=================================================
 Dataset Repeated
=================================================
=================================================
 Dataset Repeated
=================================================
=================================================
 Dataset Repeated
=================================================
=================================================
 Dataset Repeated
=================================================
Dataset shape = [1, 160, 192, 224]
Dynamic range = [0, 1.0]
Label size    = 0
Initialize data arrays
Random LayOut
tf_minibatch_np : None - Using get_minibatch_tf
(<tf.Tensor 'Dataset_1/IteratorGetNext:0' shape=(?, ?, ?, ?, ?) dtype=float16>, <tf.Tensor 'Dataset_1/IteratorGetNext:1' shape=(?, 0) dtype=float32>)
Class Conditional LayOut
160
Constructing networks...
=================================
G_Synthesis
dlatent_size
96
fmap_min
18
fmap_max
18
base_size
[5, 6, 7]
=================================
Setting up TensorFlow plugin "fused_bias_act.cu": Preprocessing... Failed!
Traceback (most recent call last):
  File "run_training.py", line 588, in <module>
    main()
  File "run_training.py", line 583, in main
    run(**vars(args))
  File "run_training.py", line 512, in run
    dnnlib.submit_run(**kwargs)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/submission/submit.py", line 343, in submit_run
    return farm.submit(submit_config, host_run_dir)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/submission/internal/local.py", line 22, in submit
    return run_wrapper(submit_config)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/submission/submit.py", line 280, in run_wrapper
    run_func_obj(**submit_config.run_func_kwargs)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/training/training_loop_3d.py", line 165, in training_loop
    G = tflib.Network('G', num_channels=training_set.shape[0], resolution=min_res, label_size=training_set.label_size, **G_args)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/network.py", line 99, in __init__
    self._init_graph()
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/network.py", line 156, in _init_graph
    out_expr = self._build_func(*self.input_templates, **build_kwargs)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/training/networks3d_stylegan2.py", line 195, in G_main
    components.synthesis = tflib.Network('G_synthesis', func_name=globals()[synthesis_func], **kwargs)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/network.py", line 99, in __init__
    self._init_graph()
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/network.py", line 156, in _init_graph
    out_expr = self._build_func(*self.input_templates, **build_kwargs)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/training/networks3d_stylegan2.py", line 425, in G_synthesis_stylegan2_3d_curated_real
    x = layer(x, layer_idx=0, fmaps=nf(1), kernel=3)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/training/networks3d_stylegan2.py", line 376, in layer
    x = modulated_conv3d_layer(x, dlatents_in[:, layer_idx], fmaps=fmaps, kernel=kernel, up=up, resample_kernel=resample_kernel, fused_modconv=fused_modconv)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/training/networks3d_stylegan2.py", line 106, in modulated_conv3d_layer
    s = apply_bias_act(s, bias_var=mod_bias_var) + 1 # [BI] Add bias (initially 1).
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/training/networks3d_stylegan2.py", line 75, in apply_bias_act
    return fused_bias_act(x, b=tf.cast(b, x.dtype), act=act, alpha=alpha, gain=gain)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/ops/fused_bias_act.py", line 68, in fused_bias_act
    return impl_dict[impl](x=x, b=b, axis=axis, act=act, alpha=alpha, gain=gain)
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/ops/fused_bias_act.py", line 122, in _fused_bias_act_cuda
    cuda_kernel = _get_plugin().fused_bias_act
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/ops/fused_bias_act.py", line 16, in _get_plugin
    return custom_ops.get_plugin(os.path.splitext(__file__)[0] + '.cu')
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/custom_ops.py", line 123, in get_plugin
    _run_cmd(_prepare_nvcc_cli('"%s" --preprocess -o "%s" --keep --keep-dir "%s"' % (cuda_file, tmp_file, tmp_dir)))
  File "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/custom_ops.py", line 73, in _run_cmd
    raise RuntimeError('NVCC returned an error. See below for full command line and output log:\n\n%s\n\n%s' % (cmd, output))
RuntimeError: NVCC returned an error. See below for full command line and output log:

nvcc "/n/holylfs05/LABS/pfister_lab/Lab/coxfs01/pfister_lab2/Lab/scajas/3DStyleGAN/dnnlib/tflib/ops/fused_bias_act.cu" --preprocess -o "/tmp/tmpwuu9uga3/fused_bias_act_tmp.cu" --keep --keep-dir "/tmp/tmpwuu9uga3" --disable-warnings --include-path "/n/home09/scajasordonez/anaconda3/envs/3dgan_tf2/lib/python3.8/site-packages/tensorflow/include" --include-path "/n/home09/scajasordonez/anaconda3/envs/3dgan_tf2/lib/python3.8/site-packages/tensorflow/include/external/protobuf_archive/src" --include-path "/n/home09/scajasordonez/anaconda3/envs/3dgan_tf2/lib/python3.8/site-packages/tensorflow/include/external/com_google_absl" --include-path "/n/home09/scajasordonez/anaconda3/envs/3dgan_tf2/lib/python3.8/site-packages/tensorflow/include/external/eigen_archive" 2>&1

/n/sw/helmod/apps/centos7/Core/gcc/12.1.0-fasrc01/bin/../libexec/gcc/x86_64-pc-linux-gnu/12.1.0/cc1plus: error while loading shared libraries: libmpfr.so.6: cannot open shared object file: No such file or directory
nvcc fatal   : Failed to preprocess host compiler properties.

